test/network_test.go

```diff
@@ -403,6 +403,7 @@ func (s *CLITestSuite) TestNetworkPrivateLinkAttachmentDescribe() {
 		{args: "network private-link attachment describe platt-111111 --output json", fixture: "network/private-link/attachment/describe-aws-json.golden"},
 		{args: "network private-link attachment describe", fixture: "network/private-link/attachment/describe-missing-id.golden", exitCode: 1},
 		{args: "network private-link attachment describe platt-invalid", fixture: "network/private-link/attachment/describe-invalid.golden", exitCode: 1},
+		{args: "network private-link attachment describe platt-azure", fixture: "network/private-link/attachment/describe-azure.golden"},
```

Comment:
we should add more test cases if we expect different output for different states

Reply:
I think the other test cases are common between aws and azure, the only thing specific to azure would be to output the alias and the id which would be covered by calling describe for an azure platt